test/python/quantum_info/operators/channel/test_chi.py

```diff
@@ -77,26 +77,26 @@ def test_evolve(self):
         # Identity channel
         chan = Chi(self.chiI)
         target_rho = np.array([[0, 0], [0, 1]])
-        self.assertAllClose(chan._evolve(input_psi), target_rho)
-        self.assertAllClose(chan._evolve(np.array(input_psi)), target_rho)
-        self.assertAllClose(chan._evolve(input_rho), target_rho)
-        self.assertAllClose(chan._evolve(np.array(input_rho)), target_rho)
+        self.assertAllClose(chan._evolve(input_psi).data, target_rho)
```

Comment:
can you avoid using private methods in the tests? There are quite a lot of them. It should be possible to write tests using the public API.

Reply:
These could be changed to `DensityMatrix(input_rho).evolve(chan)`